psi4/src/psi4/libmints/wavefunction.cc

```diff
@@ -753,6 +739,28 @@ void Wavefunction::initialize_singletons() {
     done = true;
 }
 
+const Dimension Wavefunction::doccpi(bool warn_on_beta_socc) const {
+    std::vector<int> docc_vec;
+    for (int h = 0; h < nalphapi_.n(); h++) {
+        docc_vec.push_back(std::min(nalphapi_[h], nbetapi_[h]));
+        if (warn_on_beta_socc && nbetapi_[h] > nalphapi_[h]) {
+            outfile->Printf("Warning! Irrep has more beta than alpha electrons.\n");
```

Comment:
.. in symmetry `h`

Reply:
Changed to "in symmetry `h` orbitals" instead, but done in principle.